Gordon360/Models/ViewModels/ApartmentAppIDViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Gordon360.Models.ViewModels
+{
+    public class ApartmentAppIDViewModel
+    {
+        public int AprtAppID { get; set; }
```

Comment:
Is there a reason to make this a full ViewModel instead of just using a standard `int`? It seems like in most places using the primitive type will work exactly the same way. The only exception I can think of is if you plan to add special constructor logic or methods to this ViewModel.

Reply:
The `RawSQLQuery` requires a reference type, I was actually trying to explore different options for this but I gave up for now in order to not get too sidetracked from the actual features I was supposed to be working on. 
We had actually been using that view model the entire time, I simply added it back because I couldn't figure out yet how to get it to work without wrapping it inside a `class`